doc/classes/Viewport.xml

```diff
@@ -270,10 +269,8 @@
 		<member name="gui_snap_controls_to_pixels" type="bool" setter="set_snap_controls_to_pixels" getter="is_snap_controls_to_pixels_enabled" default="true">
 			If [code]true[/code], the GUI controls on the viewport will lay pixel perfectly.
 		</member>
-		<member name="handle_input_locally" type="bool" setter="set_handle_input_locally" getter="is_handling_input_locally" default="true">
-			If [code]true[/code], this viewport will mark incoming input events as handled by itself. If [code]false[/code], this is instead done by the first parent viewport that is set to handle input locally.
-			A [SubViewportContainer] will automatically set this property to [code]false[/code] for the [Viewport] contained inside of it.
-			See also [method set_input_as_handled] and [method is_input_handled].
+		<member name="handle_input_locally" type="bool" setter="set_handle_input_locally" getter="is_handling_input_locally" default="true" is_deprecated="true">
+			[i]Obsolete.[/i] This is only kept for backward compatibility and has no effect.
```

Comment:
I think we ought to settle on a way to describe this, there's "Obsoleted" and "Deprecated" elsewhere

Reply:
I was under the impression, that "Obsoleted" is a better fit for this change, because the exposed API no longer does anything, which is aligned with how the functions in `CollisionShape3D`, `GridMap` and `ShapeCast3D` are implemented, namely as empty functions.

In contrast to "Obsoleted", the term "Deprecated" is used in the docs to indicate that the functionality is still available.
If these two terms are meant to be merged, I will make the necessary changes it in this PR.

Reply:
It looks like documenting the deprecated status has changed in the past few months. I have updated the PR accordingly.